hashchain.c

```diff
@@ -64,24 +65,37 @@ struct hash_chain hash_chain_create(void *base,
  * @param h Pointer to a hash.
  * @param tip Pointer to the "tip" hash.
  * @param hash Hash algorithm to use.
- * @param True if hash(h) == tip
+ * @param trange Maximum test size.
```

Comment:
Why would you call it `trange`?

Reply:
I think the semantics of this function is "Could the function be verified in the given range?".

Reply:
Alternatively, just use `range`.